climada/hazard/tc_tracks.py

```diff
@@ -1619,6 +1620,117 @@ def from_netcdf(cls, folder_name):
                 data.append(track)
         return cls(data)
 
+    @classmethod
+    def from_FAST(cls, folder_name: str):
+        """Create a new TCTracks object from NetCDF files generated by the FAST model, modifying
+        the xr.array structure to ensure compatibility with CLIMADA, and calculating the central
+        pressure and radius of maximum wind.
+
+        Model GitHub Repository: https://github.com/linjonathan/tropical_cyclone_risk?
+                            tab=readme-ov-file
+        Model Publication: https://agupubs.onlinelibrary.wiley.com/doi/epdf/10.1029/2023MS003686
+
+        Parameters:
+        ----------
+        folder_name : str
+            Folder name from where to read files.
+        storm_id : int
+            Number of the simulated storm
+
+        Returns:
+        -------
+        tracks : TCTracks
+            TCTracks object with tracks data from the given directory of NetCDF files.
+        """
+
+        LOGGER.info("Reading %s files.", len(get_file_names(folder_name)))
```

Comment:
Very minor suggestion,

```suggestion
        LOGGER.info(f"Reading {len(get_file_names(folder_name))} files.")
```
Prefer f-string (easier to read IMHO), or "your string {} {}".format(value_in_first_curly_braces, value_in_second_curly_braces)

https://peps.python.org/pep-0498/